setup-buildengine.sh

```diff
@@ -244,9 +284,20 @@ packVM() {
     echo "Hard linking $PWD/$OPT_VDI => $INSTALL_PATH/mer.vdi"
     ln $PWD/$OPT_VDI $INSTALL_PATH/mer.vdi
 
+    # move the docker tarball to docker directory
+    mv $INSTALL_PATH/sailfish.tar $DOCKER_INSTALL_PATH/
+    cp Dockerfile $DOCKER_INSTALL_PATH/
+
+    echo "copying $INSTALL_PATH/targets => $DOCKER_INSTALL_PATH/targets"
+    cp -R $INSTALL_PATH/targets $DOCKER_INSTALL_PATH/targets
+
     if [[ ! $OPT_NO_COMPRESSION ]]; then
         # and 7z the mersdk with chosen compression
-	7z a -mx=$OPT_COMPRESSION $PACKAGE_NAME $INSTALL_PATH/
+        7z a -mx=$OPT_COMPRESSION $PACKAGE_NAME $INSTALL_PATH/
+        pushd $DOCKER_PREFIX
+        7z a -mx=$OPT_COMPRESSION $DOCKER_PACKAGE_NAME $RELATIVE_INSTALL_PATH
```

Comment:
Using single 7z archive for both images could in theory reduce the total size by as much as 50% :)

Reply:
That would be awesome. I'll try.

Reply:
I tried, the results were less than dramatic. Having only one file would save only 1.7% space.

Reply:
Pity, it must be because the two images are too big. I tried with vdi+tar containing two JPEG images, ~6MB each, and it worked well with no special options to 7z. Then I tried with vdi+tar each containing the same JPEG images, both vdi and tar were ~400MB. With no option to 7z it produced ~800MB archive. I tried harder, based on https://stackoverflow.com/a/52771612 and I was able to produce ~400MB archive as expected. Unfortunately I wasn't able to achieve the same with our rather huge images, each ~3GB. This reminds me our intention was to use separate vdi for /srv/mer, so that tools do not need to be reinstalled when engine is updated - this could reduce build engine image sufficiently to fix this issue.